### PR TITLE
Flaky TestSQLAggregateWriteDataValues

### PR DESCRIPTION
**Pull request by tbuchaillot**
## Description

Refactor TestSQLAggregateWriteDataValues to avoid flaky tests

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). If pulling from your own
      fork, don't request your `master`!
- [ ] Make sure you are making a pull request against the **`master` branch** (left side). Also, you should start
      *your branch* off *our latest `master`*.
- [ ] My change requires a change to the documentation.
  - [ ] If you've changed APIs, describe what needs to be updated in the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] Modules and vendor dependencies have been updated; run `go mod tidy && go mod vendor`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] Check your code additions will not fail linting checks:
  - [ ] `go fmt -s`
  - [ ] `go vet`

### CODE REVIEW

File: .github/workflows/ci-test.yml [hunk 1]
```diff
@@ -81,13 +81,4 @@ jobs:
...
-          
-  sonarcloud:
-    if: ${{ always() }}
-    needs: test
-    uses: TykTechnologies/github-actions/.github/workflows/sonarcloud.yaml@main
-    with:
-      exclusions: ""
-    secrets:
-      GH_TOKEN: ${{ secrets.ORG_GH_TOKEN }}
-      SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
```
Comment (sredxny) [inline .github/workflows/ci-test.yml:93]: Why is required to remove this? @tbuchaillot

Comment (tbuchaillot) [reply .github/workflows/ci-test.yml:93]: @sredxny it's already triggered in `linter.yaml` file - and it was misconfigured.

**Review by matisidler — APPROVED** [review]

**Review by sredxny — APPROVED** [review]
lgtm

**Pull request merged**